patches/api/0085-Add-PlayerAdvancementCriterionGrantEvent.patch

```diff
@@ -58,6 +62,16 @@ index 0000000000000000000000000000000000000000..bb8d7c959cdea4b66455a49e74804ea4
 +        return cancel;
 +    }
 +
++    /**
++     * Gets the AdvancementProgress.
```

Comment:
I would clarify that this is the advancement progress **with** the criteron granted. This event kinda breaks the norm by having the change happen, then fire the event and undo the change if it was cancelled. Maybe that should be changed, but that can be for another time.